fdbserver/ClusterController.actor.cpp

```diff
@@ -2320,7 +2322,7 @@ ACTOR Future<Void> startEncryptKeyProxy(ClusterControllerData* self) {
 
 ACTOR Future<Void> monitorEncryptKeyProxy(ClusterControllerData* self) {
 	state EncryptionAtRestMode encryptMode = wait(self->encryptionAtRestMode.getFuture());
-	if (!encryptMode.isEncryptionEnabled()) {
+	if (!encryptMode.isEncryptionEnabled() && !SERVER_KNOBS->ENABLE_REST_KMS_COMMUNICATION) {
```

Comment:
Can you please add a TraceEvent here:
`TraceEvent(SevInfo, "EKPNotConfigured")`

`SevInfo` is OK as it logs **only** during CC startup